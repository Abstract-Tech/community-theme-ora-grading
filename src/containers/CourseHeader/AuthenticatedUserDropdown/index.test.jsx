import React from 'react';
import { shallow } from 'enzyme';

import { AuthenticatedUserDropdown } from '.';

jest.mock('@edx/frontend-platform', () => ({
  getConfig: () => ({
    SUPPORT_URL: '<SUPPORT_URL>',
  }),
}));
jest.mock('@edx/paragon', () => ({
  Dropdown: () => 'Dropdown',
}));
jest.mock('./UserAvatar', () => 'UserAvatar');
jest.mock('./UserMenu', () => 'UserMenu');

describe('Header AuthenticatedUserDropdown component', () => {
  const props = {
    intl: { formatMessage: (msg) => msg.defaultMessage },
    username: 'test-username',
  };
  test('snapshot', () => {
    expect(
      shallow(<AuthenticatedUserDropdown {...props} />),
    ).toMatchSnapshot();
  });
});
