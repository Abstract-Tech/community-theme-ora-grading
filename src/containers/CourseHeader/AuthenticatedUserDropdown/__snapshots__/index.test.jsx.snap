// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header AuthenticatedUserDropdown component snapshot 1`] = `
<Fragment>
  <a
    className="text-gray-700 mr-3"
    href="<SUPPORT_URL>"
  >
    Help
  </a>
  <Dropdown
    className="user-dropdown"
  >
    <UserAvatar
      username="test-username"
    />
    <UserMenu
      username="test-username"
    />
  </Dropdown>
</Fragment>
`;
